File: JAVA/loop/factorial.java
import java.util.Scanner;

public class factorial {
    public static void main(String[] args) {
        int no, fact = 1;

        Scanner sd = new Scanner(System.in);

        System.out.println("Enter a number = ");
        no = sd.nextInt();

        for (int i = 2; i <= no; i++) {
            fact *= i;
        }

        System.out.println("Factorial of " + no + " is " + fact);
    
        sd.close();
    }
}

File: C++/Class/factorial.cpp
#include <iostream>
using namespace std;

int main()
{
    int no, i, fact = 1;
    cout << "Enter number = ";
    cin >> no;

    for (i = 2; i <= no; i++)
    {
        fact *= i;
    }
    cout << "Factorial = " << fact;
    return 0;
}